_topic_map.yml

```diff
@@ -2407,12 +2393,21 @@ Topics:
     File: serverless-apiserversource
   - Name: Using PingSource
     File: serverless-pingsource
-# Metering
-- Name: Using metering with OpenShift Serverless
-  File: serverless-metering
+  # Sinkbinding
+  - Name: Using SinkBinding
+    File: serverless-sinkbinding
 # Integrations
 - Name: Integrations
   Dir: integrations
   Topics:
   - Name: Using NVIDIA GPU resources with serverless applications
     File: gpu-resources
+# Release notes
+- Name: Release Notes
```

Comment:
I know it was like this before it was moved, but "Release Notes" really should be "Release notes". Same w/ the name of the assembly.